new-pipeline/src/dashboards.js

```diff
@@ -1081,6 +1081,23 @@ function getDescriptionLink(metric, channel, description) {
       class: "btn btn-outline-primary fa fa-info-circle",
     }).text(" More details"));
   }
-
   return link;
 }
+
+function getUseCounterLink(metric, channel, description) {
+  if (metric.startsWith("USE_COUNTER") !== true) {
+    return null;
+  }
+  else {
+    metricSplit = metric.split("_");
```

Comment:
Add a `var`, otherwise `metricSplit` is a [global variable](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/var#Description).